templates/new_service/.github/workflows/build-nocache.yml

```diff
@@ -0,0 +1,45 @@
+name: Build No Cache
+
+on:
+  workflow_dispatch:
+
+  schedule:
+    - cron: '30 12 * * 0'
```

Comment:
Can we add a comment to explain when it will happen?

Reply:
added comment